.travis.yml

```diff
@@ -1,6 +1,5 @@
-dist: trusty
-sudo: false
-language: generic
+dist: xenial
```

Comment:
I wonder if we could just  remove the `dist`, since we always use the default value ?

Reply:
:thinking: Let me try